edisgo/flex_opt/check_tech_constraints.py

```diff
@@ -364,176 +422,301 @@ def _station_load(edisgo_obj, grid):
     Returns
     -------
     :pandas:`pandas.DataFrame<DataFrame>`
-        Dataframe containing over-loaded stations, their missing apparent
-        power at maximal over-loading and the corresponding time step.
-        Index of the dataframe are the representatives of the grids with
-        over-loaded stations. Columns are 's_missing' containing the missing
-        apparent power at maximal over-loading in MVA as float and 'time_index'
-        containing the corresponding time step the over-loading occured in as
-        :pandas:`pandas.Timestamp<Timestamp>`.
+        In case there are no over-loading problems returns an empty dataframe.
+        In case of over-loading problems the dataframe contains the name of the
+        over-loaded station (grid's name with the extension '_station') in the index.
+        Columns are 's_missing' containing the missing apparent power at maximal
```

Comment:
The reason is that it is easier for the reinforcement measures. In case of lines the parallel lines always have the same line type so from the relative overload it is easy to determine how many parallel lines are needed. In case of transformers they don't have to be of the same type and the smallest transformer that will solve the overloading issue is used. Therefore it is easier to not have the relative overload but the additional apparent power that is needed. Since these overload functions are written specifically for the grid reinforcement, I would keep it like that. Maybe we could keep those specific functions in check_tech_constraints and move the more general functions to tools?